lib/react_on_rails/utils.rb

```diff
@@ -208,6 +213,13 @@ def self.react_on_rails_pro_version
                                     end
     end
 
+    def self.rsc_support_enabled?
+      return false unless react_on_rails_pro?
+
+      rorp_config = ReactOnRailsPro.configuration
+      rorp_config.respond_to?(:enable_rsc_support) && rorp_config.enable_rsc_support
```

Comment:
Maybe worth caching (especially with `respond_to?` involved)? Something like
```suggestion
      return @rsc_support_enabled if defined?(:rsc_support_enabled)
      return false unless react_on_rails_pro?

      rorp_config = ReactOnRailsPro.configuration
      @rsc_support_enabled = rorp_config.respond_to?(:enable_rsc_support) && rorp_config.enable_rsc_support
```